lib/puppetserver/ca/utils/file_system.rb

```diff
@@ -14,13 +14,9 @@ def self.write_file(*args)
           instance.write_file(*args)
         end
 
-        def self.ensure_dir(setting)
-          instance.ensure_dir(setting)
-        end
-
-        def self.ensure_file(location, content, mode)
-          if !File.exist?(location)
-            instance.write_file(location, content, mode)
+        def self.ensure_dirs(one_or_more_settings)
```

Comment:
^^ removed ensure_file as I removed the only place we ever called it in the import.rb

ensure_dirs now takes multiple settings.

Reply:
Can we call them "dirs" instead of "settings"? Even though we always pass in paths from settings, this method could definitely work on any file path.